build/js/intlTelInput.js

```diff
@@ -636,11 +644,27 @@ https://github.com/Bluefieldscom/intl-tel-input.git
         },
         // decide where to position dropdown (depends on position within viewport, and scroll)
         _setDropdownPosition: function() {
-            var inputTop = this.telInput.offset().top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
-            dropdownFitsBelow = inputTop + this.telInput.outerHeight() + this.dropdownHeight < windowTop + $(window).height(), dropdownFitsAbove = inputTop - this.dropdownHeight > windowTop;
+            var showDropdownContainer = this.options.dropdownContainer && !this.isMobile;
+            if (showDropdownContainer) this.dropdown.appendTo(this.options.dropdownContainer);
+            var that = this, pos = this.telInput.offset(), // show the menu and grab the dropdown height
+            dropdownHeight = this.countryList.removeClass("hide").outerHeight(), inputTop = pos.top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
```

Comment:
Hmm but if there's no dropdownContainer, then countryList is already in the DOM and if you do removeClass("hide") then it will be visible! You need to addClass("v-hide") first to get the height. Also we only need to do this once, so I recommend checking if (this.dropdownHeight) is set, and if not, doing addClass("v-hide").removeClass("hide") and setting this.dropdownHeight and then doing addClass("hide").removeClass("v-hide") again. This will only need to be done once... What do you think?